registries/_format/decimal.md

```diff
@@ -0,0 +1,31 @@
+---
+owner: baywet
+issue: 889
+description: A fixed point decimal number
+base_type: string
+layout: default
+remarks: Potential loss of precision when used with type number. Not specific enough about the size of the integral and fraction parts without the use of extensions.
+---
+
+# <a href="..">{{ page.collection }}</a>
+
+## {{ page.slug }} - {{ page.description }}
+
+Base type: `{{ page.base_type }}`.
+
+The `{{page.slug}}` format represents a fixed point decimal number.
+
+{% if page.issue %}
+### GitHub Issue
+
+* [#{{ page.issue }}](https://github.com/OAI/OpenAPI-Specification/issues/{{ page.issue }})
+{% endif %}
+
+{% if page.remarks %}
+### Remarks
+
+When the decimal format is used in combination with the number type, unintentional loss of precision can happen during serialization as most JSON serializers will serialize the value **1.10** to **1.1**.
```

Comment:
Removing trailing fractional zeroes doesn't strike me as loss of precision.

More problematic is parsing/deserialization: 9999999999999999 becomes 10000000000000000 in JavaScript.

Reply:
Other than that I like the new proposal

Reply:
it's a problem for scientific measurements and other things that depend on it. But I think what I've been meaning here is actually [accuracy](https://en.wikipedia.org/wiki/Accuracy_and_precision) (again my math was taught to me in French, and I butcher things in English all the time)

Reply:
reverted to the default template for alignment, and because we're not mentioning the loss of accuracy anymore, which is a JSON specific issue. (other serialization formats could be used)

Reply:
> (other serialization formats could be used)

oh, good point!